pkg/kdk/ssh.go

```diff
@@ -21,11 +21,44 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/codeskyblue/go-sh"
+	"github.com/docker/docker/api/types"
 )
 
-func Ssh(cfg KdkEnvConfig, logger logrus.Entry) error {
+func Ssh(cfg KdkEnvConfig, logger logrus.Entry) {
 	logger.Info("Connecting to KDK container")
 
+	// Pull KDK image
```

Comment:
Does this belong in the ssh.go file / ssh function?  Seems like an init thing (following example of vagrant)

Reply:
intent was to minimize the number of commands required to get the KDK running.
so  `kdk init && kdk pull && kdk up && kdk ssh` -> `kdk init && kdk ssh`